packages/tables/src/Filters/Indicator.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Filament\Tables\Filters;
+
+use Closure;
+use Filament\Support\Components\Component;
+use Filament\Support\Concerns\HasColor;
+
+class Indicator extends Component
+{
+    use HasColor;
+
+    protected bool | Closure $isRemovable = true;
+
+    protected string | Closure | null $label = null;
+
+    protected string | Closure | null $removeField = null;
+
+    protected string | Closure | null $removeLivewireClickHandler = null;
+
+    public function __construct(string $label)
```

Comment:
Am I missing something, or shouldn't the type be changed from `string` to `string | Closure | null`? The same applies to `label()`, including the evaluation of `$label` in `getLabel()`.

Reply:
Good point, fixed that on the 3.x branch

Reply:
Yup, thanks for noticing and fixing.